docs/inquiries/commercial-inquiries.md

```diff
@@ -0,0 +1,6 @@
+Commercial inquiries
+--------------------
+
+If you have any questions about our services feel free to contact us using
```

Comment:
Any news regarding the live chat possibility?

Reply:
@pietrushnic Yes, "live chat using mattermost" sounds great. My other ideas _(not about live chat but Dasharo.com in general)_ - have been mentioned at https://chat.3mdeb.com/team-3mdeb/pl/1ta8zu9ghin89gj1x17rxf3etc and I hope there's some action

Reply:
@mikebdp2 Other suggestions will be addressed with the outsourced web development team in April. Live chat will first be connected to Dasharo, then maybe to 3mdeb if it will work.

Reply:
Of course then we have to update documentation.